ckanext/nhs/templates/home/layout1.html

```diff
@@ -1,64 +1,34 @@
-<div class="container searchbar" id="content" role="search">
-  <div class="row">
-    <div class="col-lg-7 col-md-9">
-      <div class="block">
-        <h1>Search Datasets</h1>
-        {% block search %}
-          {% snippet 'home/snippets/search.html' %}
-        {% endblock %}
-      </div>
+<main>
+    <div class="survey" id="survey">
+        {% block survey%} {% snippet 'home/snippets/feedback.html' %} {% endblock %}
+    </div>
+    <div class="hero" role="search">
+        {% block hero%} {% snippet 'home/snippets/hero.html' %} {% endblock %}
     </div>
-  </div>
- 
-</div>
 
-  <div role="main">
-    <div class="container">
-      <div class="row row2">
-        <div class="col-md-11 col1">
-          {# Note: this featured_group block is used as an example in the theming
-             tutorial in the docs! If you change this code, be sure to check
-             whether you need to update the docs. #}
-          {# Start template block example. #}
-        </div>
-        <div class="col-md-11 col2">
-          <h2 class="section-heading" id="new-datasets">Latest Updates</h2>
+    <div class="themes">
+        {%block themes %} {% snippet 'home/snippets/themes.html' %} {% endblock %}
+    </div>
+    <div class="latest-info">
+        {%block latest_info %} {% snippet 'home/snippets/latest_info.html'%} {% endblock %}
+    </div>
+    <div class="newsletter">
+        {%block newsletter %} {% snippet 'home/snippets/newsletter.html'%} {% endblock %}
+    </div>
+</main>
 
-          <div class="custom-card m-b-60">
-            <!-- Resources -->
-            {% for resource in h.get_latest_resources() %}
-              <div class="single-list">
-                <a href="/dataset/{{resource['package_id']}}/resource/{{resource['id']}}">
-                {% if resource['extras']['title'] %}
-                  {% set resouce_title = resource['extras']['title'] %}
-                {% elif resource['name'] %}
-                  {% set resouce_title = resource['name'] %}
-                {% else %}
-                  {% set resouce_title = 'Untitled resource' %}
-                {% endif %}
-                {{ resouce_title }}
-                </a>
-                {% set resource_datetime = resource['created'] %}
-                {% if resource['last_modified'] %}
-                  {% set resource_datetime = resource['last_modified'] %}
-                {% endif %}
-                <p> <small> {% snippet 'snippets/local_friendly_datetime.html', datetime_obj=resource_datetime %}</small></p>
-                <p>{{h.render_markdown(resource['description'])}}</p>
-              </div>
-            {% endfor %}
-          </div>
+<script>
+    window.onscroll = function() {
+        myFunction()
```

Comment:
Minor but can we add a related script inside on scroll event function? eg.

```suggestion
           var header = document.getElementById("survey");
           var sticky = header.offsetTop;
           if (window.pageYOffset > sticky) {
              header.classList.add("sticky");
           } else {
              header.classList.remove("sticky");
          }
```

Reply:
@sagargg The structure of this script is important. Structuring it this way results in a sort of "endless refresh" of the banner when scrolled. Please see video below
https://www.loom.com/share/ac13468003224463a1af1682e9f30d1f